packages/cf-style-const/src/variables.js

```diff
@@ -60,5 +71,6 @@ export default {
   colorPurpleLight: lighten(colorOffsetLight, '#d91698'),
   colorPurpleDark: darken(colorOffsetDark, '#d91698'),
   colorTwitterBlue: '#00aced',
-  colorFacebookBlue: '#3b5998'
+  colorFacebookBlue: '#3b5998',
+  mainBackground: '#e6e6e6'
```

Comment:
Sure thing. It's always the way with naming colour variables, haha. Maybe go with `colorMainBackground: colorLightishGray, colorLightishGray: '#e6e6e6'` or similar? Just that if someone wants `#e6e6e6` they'll probably start using `colorMainBackground` for their shadows, haha.